content/en/events/2024/kcsna/social.md

```diff
@@ -9,6 +9,8 @@ There will be food, live music, activities, and more!
 
 *The contributor social is _not_ a public event.  You must be [registered] in advance to attend the Contributor Summit in order to attend the Social, and must be a Kubernetes contributor.  Community members who are not registered and show up at the Social will be turned away.*
 
+**You must bring your Kubecon badge to get into the Social.**
```

Comment:
Extreme nit: KubeCon

Feel free to ignore, but I have a compulsion to point it out. :D